pom.xml

```diff
@@ -52,12 +52,19 @@
     </profile>
     <profile>
       <id>2020-09</id>
+      <properties>
+        <platform-release>2020-09</platform-release>
+        <orbit-build>S20200817205214</orbit-build>
```

Comment:
BTW I should have mentioned this - the reason I added the new one is that this didn't build anymore because of the reference to the S-build here.